tests/test_references.py

```diff
@@ -273,4 +278,7 @@ def test_register_markdown_anchors() -> None:
         "alias5": "page#alias5",
         "alias6": "page#alias6",
         "alias7": "page#alias7",
+        "alias8": "page#alias8",
+        "alias9": "page#heading-more2",
+        "alias10": "page#alias10",
     }
```

Comment:
I have a few more tests in mind to add, will send a commit soon

Reply:
More tests are here
https://github.com/oprypin/mkdocs-autorefs/commit/cc7ab04767ba46145a1cd0d9695d9badb32c4bf4#diff-52b8f2e623489bd3570119271e0622f9b5adc44eb98461662905000fa7972ce1